IaC/PnP/monitoring/aks/online/aks_secure_baseline/configuration/networking/private_dns.tfvars

```diff
@@ -0,0 +1,24 @@
+
+private_dns = {
+  dns1 = {
+    name               = "ngsa-monitoring-dev.com"
```

Comment:
Is there an easy way to find all instances of the DNS record that need to be updated throughout the code?

Reply:
not currently, in some instances the domain is embedded in some resource names as well.  A better structured method should be implemented when the Deploy team adjusts the baseline.